commands-yml/validator.js

```diff
@@ -11,41 +12,41 @@ validate.validators.hasAttributes = function (value, options) {
     return;
   }
 
-  if (typeof value === 'object' && !Array.isArray(value)) {
+  if (_.isObject(value) && !_.isArray(value)) {
```

Comment:
what should be checked here? maybe you want to apply _isPlainObject_ check?

Reply:
We're expecting either an object or an array of objects. If it's just an object, I want to change it to an array of one. 

Does `isPlainObject` do the same thing as `_.isObject(value) && !_.isArray(value)`?

Reply:
`!_.isArray(value)` should suffice